enhancements/kube-apiserver/audit-policy.md

```diff
@@ -331,6 +363,7 @@ enhancement:
   when this feature is used?
 - Will any other components on the node change? For example, changes to CSI, CRI
   or CNI may require updating that component before the kubelet.
+- How can Audit Policies made to be made more dynamic in the future?
 
 ## Implementation History
```

Comment:
we need a discussion of the alternate, more extensive solution of having another field that is orthorgonal to `policy`, e.g.:

```
policy: Default | WriteRequestBodies | AllRequestBodies
scope: All | OauthAuthenticated
```